include/psp2/ctrl.h

```diff
@@ -191,6 +218,13 @@ int sceCtrlSetActuator(int port, const SceCtrlActuator* pState);
  */
 int sceCtrlSetLightBar(int port, SceUInt8 r, SceUInt8 g, SceUInt8 b);
 
+/**
+ * Get controller port information.
+ *
+ * @param[out] info - Return value, use char[16] buffer
+ * @return 0, <0 on error
+ */
+int sceCtrlGetControllerPortInfo(void *info);
```

Comment:
Any more information for this function? I understand from the name that it gets the controller port information but can you clarify what this information is? stuff like format, etc